hedera-node/src/main/java/com/hedera/services/state/logic/AwareNodeDiligenceScreen.java

```diff
@@ -32,17 +32,21 @@
 
 import static com.hedera.services.txns.diligence.DuplicateClassification.NODE_DUPLICATE;
 import static com.hedera.services.utils.EntityIdUtils.readableId;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.ACCOUNT_ID_DOES_NOT_EXIST;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.DUPLICATE_TRANSACTION;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_NODE_ACCOUNT;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_PAYER_SIGNATURE;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.INVALID_TRANSACTION_DURATION;
 import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.OK;
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.PAYER_ACCOUNT_DELETED;
 
 public class AwareNodeDiligenceScreen {
 	private static final Logger log = LogManager.getLogger(AwareNodeDiligenceScreen.class);
 
 	final static String WRONG_NODE_LOG_TPL = "Node {} (member #{}) submitted a txn meant for node account {} :: {}";
 	final static String MISSING_NODE_LOG_TPL = "Node {} (member #{}) submitted a txn w/ missing node account {} :: {}";
+	final static String MISSING_PAYER_ACCOUNT_LOG = "Node {} (member #{}) submitted a txn with missing payer account {} :: {}";
```

Comment:
We probably don't need to log anything in the case of a missing or deleted payer account.

A well-behaved node will **never**, under any circumstances, submit a transaction with a missing or non-designated node account. So if this ever happened, it would deserve immediate attention, and a `WARN` log is the fastest way to get someone's attention. :)

But even a well-behaved node could get "unlucky" and submit a transaction `T` whose payer account ends up deleted by the time that `T` reaches consensus. (Your EET test demonstrates how this can happen, in fact!)

Reply:
I removed the warning logs. Thanks for the feedback!